app/src/main/java/com/android/sample/receivers/NotificationReceiver.kt

```diff
@@ -0,0 +1,29 @@
+package com.android.sample.receivers
+
+import android.app.NotificationManager
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import androidx.core.app.NotificationCompat
+import com.android.sample.R
+
+class NotificationReceiver : BroadcastReceiver() {
+  override fun onReceive(context: Context, intent: Intent) {
+    val activityId = intent.getIntExtra("activityId", 0)
```

Comment:
Should not fallback to 0. Should rather discard the request.

Reply:
Changes made in new code!